### PR TITLE
Refactored README.md files

### PR DESCRIPTION
**Pull request by Palych18**

### CODE REVIEW

File: functions/getMaxNumber/getMaxNumber.test.js
Comment (nagorapro) [inline functions/getMaxNumber/getMaxNumber.test.js:1]: Format

File: functions/getMinNumber/getMinNumber.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * @function getMinNumber
+ * @param {number[]} numbers
+ * @returns {number | undefined}
+ */
+
+export const getMinNumber = (numbers) => {
+  if (!numbers.every((numbers) => Number.isInteger(numbers))) return undefined;
```
Comment (nagorapro) [inline functions/getMinNumber/getMinNumber.js:8]: Format long line

File: functions/getMinNumber/getMinNumber.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  const minNumber = Math.min(...numbers);
+
+  return minNumber;
```
Comment (nagorapro) [inline functions/getMinNumber/getMinNumber.js:12]:
```suggestion
  return Math.min(...numbers);
```

File: functions/getMinNumber/getMinNumber.test.js
Comment (nagorapro) [inline functions/getMinNumber/getMinNumber.test.js:1]: Format

File: functions/identifyLoveFlower/identifyLoveFlower.test.js
Comment (nagorapro) [inline functions/identifyLoveFlower/identifyLoveFlower.test.js:1]: Format

**Review by nagorapro — COMMENTED** [review]
Need fix

File: functions/getMaxNumber/getMaxNumber.js
Comment (nagorapro) [inline functions/getMaxNumber/getMaxNumber.js:1]: Format condition

File: functions/getMaxNumber/getMaxNumber.test.js
Comment (nagorapro) [inline functions/getMaxNumber/getMaxNumber.test.js:1]: Format

**Review by nagorapro — COMMENTED** [review]
Need fixed

File: functions/getMaxNumber/getMaxNumber.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { getMaxNumber } from './getMaxNumber.js';
+
+describe('getMaxNumber:', () => {
+  it('Unit test 1', () => {
+    const valuesNum = getMaxNumber(['5', 5]);
```
Comment (nagorapro) [inline functions/getMaxNumber/getMaxNumber.test.js:5]:
```suggestion
    const maxNum = getMaxNumber(['5', 5]);
```

File: functions/getMinNumber/getMinNumber.test.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { getMinNumber } from './getMinNumber.js';
+
+describe('getMinNumber:', () => {
+  it('Unit test 1', () => {
+    const valuesNum = getMinNumber(['5', 5]);
```
Comment (nagorapro) [inline functions/getMinNumber/getMinNumber.test.js:5]:
```suggestion
    const minNum = getMinNumber(['5', 5]);
```

**Review by nagorapro — COMMENTED** [review]
Need fix

File: functions/getSum/getSum.test.js
Comment (nagorapro) [inline functions/getSum/getSum.test.js:1]: Format

**Review by nagorapro — COMMENTED** [review]
Need fix

File: functions/getMinNumber/getMinNumber.test.js (hunk 4 above)
Comment (Palych18) [reply functions/getMinNumber/getMinNumber.test.js:5]: Test

**Review by nagorapro — APPROVED** [review]

**Pull request merged**